src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -510,7 +510,7 @@ Domain<3> BinaryCompactObject::create_domain() const {
   std::unordered_map<std::string, ExcisionSphere<3>> excision_spheres{};
   if (object_A_.is_excised()) {
     excision_spheres.emplace(
-        "ObjectAExcisionSphere",
+        "ExcisionSphereA",
```

Comment:
Maybe replace all these hardcoded names with `get_output(label)`? It's easier to regex-replace `domain::ObjectLabel::A` than literal single-character `A`s :)

Reply:
I was going to do a separate PR where I used the `ObjectLabel` for the names in the BBH domains (excision spheres and functions of time). Does that sound ok?

Reply:
sure whatever is easiest